packages/core/src/textures/Texture.ts

```diff
@@ -105,6 +118,12 @@ export class Texture<R extends Resource = Resource> extends EventEmitter
      */
     public defaultAnchor: Point;
 
+    /**
+     * Default width of the non-scalable border that is used if 9-slice plane is created with this texture.
+     * (See {@link PIXI.NineSlicePlane})
+     */
+    public defaultNineSliceBorders?: INineSliceBorders;
```

Comment:
Same API-naming preference here, decouple from NineSlicePlane.

```suggestion
    public defaultBorders?: ITextureBorders;
```

Reply:
Is there a reason why we wouldn't use a Rectangle here like the other Texture properties (trim, orig, frame, etc)

Reply:
Internally in TexturePacker we do indeed store the "inner rectangle" instead of border widths, with leftBorder=innerRect.x, rightBorder=sprite.width-innerRect.width-innerRect.y and so on. As PixiJS expects borders and other frameworks like Unity also use border values to configure 9-slice scaling, we thought that it's better to stay with borders.